autogluon/inference/docker/0.6/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -58,5 +58,1164 @@
       "title": "CVE-2022-45907 - torch",
       "reason_to_ignore": "patch via AWS binaries in parent container"
     }
+  ],
```

Comment:
please provide reason_to_ignore for all skips

Reply:
added